staging/nvidia/charts/nvidia-driver/values.yaml

```
@@ -0,0 +1,51 @@
+# Default values for nvidia-driver.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: "nvidia/driver"
+  tag: "418.40.04-ubuntu16.04-aws"
+  pullPolicy: "IfNotPresent"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hosts:
+    - host: chart-example.local
+      paths: []
+
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources: {}
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+nodeSelector:
+  konvoy.mesosphere.com/gpu-provider: NVIDIA
+
+tolerations: []
+
+affinity: {}
```

Again here I think we should just support initContainers on day 1 as it can add a lot of value for us and end-users right out of the box:
```suggestion
affinity: {}

initContainers: []
```

And it would need to be added to the `Daemonset` here too.